### PR TITLE
Cirrus: Install conmon in Fedora VMs

### PR DESCRIPTION
**Pull request by cevich**

### CODE REVIEW

File: contrib/cirrus/packer/fedora_setup.sh [hunk 1]
```diff
@@ -31,6 +31,7 @@ ooe.sh sudo dnf install -y \
     bridge-utils \
     btrfs-progs-devel \
     bzip2 \
+    conmon \
```
Comment (lsm5) [inline contrib/cirrus/packer/fedora_setup.sh]: So, older versions of podman didn't require the package, but conmon was still installed as part of the main podman package itself at `/usr/libexec/podman/conmon`. Just making sure we're on the same page.

Comment (cevich) [reply contrib/cirrus/packer/fedora_setup.sh]: oh yes, and thanks for the reminder, there's a TODO I should also strip out...

Comment (cevich) [reply contrib/cirrus/packer/fedora_setup.sh]: ...done.

**Comment by cevich** [comment]
Ref: added commit to remove test_crun_task

    [14:11] <giuseppe> cevich, baude yeah that is a regression in 0.10.1
    [14:12] <giuseppe> we need crun 0.10.2
    [14:13] <cevich> giuseppe: oh you're here, great.  Okay if I kill that task or would u prefer I just 
    comment out the 'dnf install -y crun' or something?
    [14:13] <giuseppe> cevich, we can disable it for now
    [14:13] <cevich> giuseppe: kk. thx.

**Comment by cevich** [comment]
@baude @mheon PTAL  Fresh VM images w/ conmon package and fix for (currently) broken upstream crun.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**